modules/db/variables.tf

```diff
@@ -0,0 +1,60 @@
+variable "resource_group" {
+  description = "Azure resource group."
+  type        = string
+  default     = "aap_on_azure"
+}
+
+variable "location" {
+  description = "Azure location."
+  type        = string
+  default     = "East US"
+}
+
+variable "deployment_id" {
+  description = "Creates a random string that will be used in tagging for correlating the resources used with a deployment of AAP."
+  type    = string
+  validation {
+    condition     = (length(var.deployment_id) == 8 || length(var.deployment_id) == 0) && (can(regex("^[a-z]", var.deployment_id)) || var.deployment_id == "")
+    error_message = "deployment_id length should be 8 chars and should contain lower case alpha chars only"
+  }
+}
+
+variable "infrastructure_db_username" {
+  description = "PostgreSQL username."
+  type        = string
+  default     = "psqladmin"
+}
+
+variable "infrastructure_db_password" {
+  description = "PostgreSQL password."
+  type        = string
+  default     = "ChangeMe1234"
```

Comment:
Defaulted password here.

Is it normal to have all the variables repeated? Seems like there'd be a better way than declaring them twice.

Reply:
```suggestion
  sensitive = true
```

Reply:
Good catch, I forgot I had the variable in a module too!

As far as proper terraform use of variables, I only know what I have read and tried so far.  Not sure if there is another way to do this type of thing.